### DISCUSSION TITLE
"Lite" mode for only using mouse/keyboard control

### DISCUSSION THREAD

**Discussion by jonatino** [rustdesk/rustdesk#6915] [Ideas]
Would love a mode for rustdesk for when I just need to use my phone as a remote mouse/keyboard but not view the screen/all the extra bells and whistles rustdesk offers.

This mode will greatly reduce battery usage, and would be great at not interrupting anything on screen. It will allow for quick adjustments like playing/pausing a movie, quickly changing volume, etc.

**Comment by jonatino** [comment]
For ideas look at https://www.remotemouse.net/

Just some basic function that turns your phone into a touchpad input device.

**Comment by jgrussell** [comment]
I would also appreciate this functionality not only for controlling Android devices but also Windows and Linux machines that are physically sitting on my desk.
